app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -496,6 +546,32 @@ const CallScreen = ({componentId, currentCall, participantsDict, teammateNameDis
                 </View>
                 {usersList}
                 {screenShareView}
+                {micPermissionsError &&
+                    <View style={style.errorWrapper}>
+                        <Pressable onPress={goToSettings}>
+                            <View style={style.errorBar}>
+                                <CompassIcon
+                                    name='microphone-off'
+                                    style={[style.errorIcon, {paddingRight: 9}]}
```

Comment:
same comment as in the other component, which brings me to believe that this entire block can be its own component and use it in both places, thoughts?

Reply:
🤦 You're totally right, can't believe I missed that. This was a little rushed, apologies for all the mistakes. Hopefully everything is fixed now.

Reply:
No need to apologize, that is why we do reviews :)

Thanks for making the changes